docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,176 @@
+    0.8548237161075367,
+    0.5357984508167614],
+   'properties': {'score': 0.873895525932312,
+    'image_size': [1324, 1003],
+    'image_mode': 'RGB',
+    'image_format': None,
+    '_element_index': 1,
+    'page_number': 2},
+   'binary_representation': ...
+ }
+```
+
+Above, you can see the JSON output from the call to Aryn DocParse. The output is a JSON object with a `status` field that shows the status of the call and an `elements` field that contains a list of elements extracted from the document. Each element has a `type`, `bbox`, `properties`, and `text_representation` field. The `type` field indicates the type of the element (e.g. text, image, table, etc.), the `bbox` field contains the coordinates of the bounding box around the element, the `properties` field contains additional information about the element (e.g. confidence score, page number, etc.), and the `text_representation` field contains the text content of the element. The call ID has been redacted for privacy reasons.
+
+## Examining Individual Elements
+
+### Extracting an Image from the Document
+
+```json image.json
+
+{ 
+  type': 'Image',
+ 'bbox': [0.0873676793715533,
+  0.08911955399946733,
+  0.8548237161075367,
+  0.5357984508167614],
+ 'properties': {'score': 0.873895525932312,
+  'image_size': [1324, 1003],
+  'image_mode': 'RGB',
+  'image_format': None,
+  '_element_index': 1,
+  'page_number': 2},
+ 'binary_representation': ...
+}
+```
+The first element we see is an image. We get back a bounding box and a binary representation of the image, which we can use to display the image. The bouning boxes are given in the format [x1, y1, x2, y2], where (x1, y1) is the top-left corner and (x2, y2) is the bottom-right corner of the bounding box.
+
+### Extracting Captions from the Document
+
+```json caption.json
+
+{'type': 'Caption',
+ 'bbox': [0.08805406458237591,
+  0.5466218150745739,
+  0.8386106244255515,
+  0.5634380548650568],
+ 'properties': {'score': 0.6033732891082764,
+  '_element_index': 3,
+  'page_number': 2},
+ 'text_representation': 'Figure 1. Accident airplane as it came to rest (Source: Federal Aviation Administration)\n'}
+```
+The second element we see is a caption. We get back a bounding box and the text content of the caption.
+
+### Extracting Text from the Document
+
+```json text.json
+{'type': 'Text',
+ 'bbox': [0.08857761158662684,
+  0.5770943936434659,
+  0.5373964197495404,
+  0.5928012917258523],
+ 'properties': {'score': 0.589328944683075,
+  '_element_index': 5,
+  'page_number': 2},
+ 'text_representation': 'The wreckage was retained for further examination. \n'}
+```
+The third element we see is a text element. We get back a bounding box and the text content of the element.
+
+### Extracting a Table from the Document
+
+```json table.json
+
+{ 
+ 'type': 'table',
+ 'bbox': [0.09319061279296875,
+  0.6873569003018466,
+  0.9122876694623162,
+  0.8174545010653409],
+ 'properties': {'score': 0.8106631636619568,
+  'title': None,
```

Comment:
Does every table have a 'title' field? What does it signify?

Reply:
It represents the title tag of the table, which we might populate from the HTML representation.